docs/source/considerations.rst

```diff
@@ -108,3 +108,27 @@ however you may notice some differences:
    to. As the binding was executed on initialization, we need to use the
    ``addr()`` method, which will return the address associated to the alias
    passed as parameter (in the example above it is ``main``).
+
+
+Setting initial attributes
+==========================
+
+Many times, after spawning an agent, we want to set some attributes, which
+may be used to configure the agent before it starts working with the rest of
+the multi-agent system:
+
+.. code-block:: python
+
+   a0 = run_agent('foo')
+   a0.set_attr(x=1, y=2)
+
+It is such a common task that a parameter ``init`` can be used when running
+the agent for exactly that:
+
+.. code-block:: python
+
+   a0 = run_agent('foo', init=dict(x=1, y=2))
```

Comment:
Now that I think about it, do you prefer that syntax? I personally prefer `init={'x': 1, 'y': 2}`, but its up to you 👍